core/src/main/res/layout/activity_dev_menu.xml

```diff
@@ -25,46 +25,101 @@
         android:layout_height="wrap_content"
         android:background="@android:color/background_light"
         android:orientation="vertical"
-        android:layout_alignParentBottom="true"
-        android:padding="15dp">
+        android:layout_alignParentBottom="true">
 
-        <TextView
-            android:id="@+id/title_text_view"
+        <RelativeLayout
```

Comment:
I thought about using a Toolbar, but it requires a minimum of SDK 21, so I chose to implement my own header to avoid assuming the customer has at least SDK 21.

Reply:
Hmm, right, `android.widget.Toolbar` does have that limitation.  I think you should be able to use  [`androidx.appcompat.widget.Toolbar`](https://developer.android.com/reference/androidx/appcompat/widget/Toolbar) though, as it is backported for older APIs.

To use, you would just add this in your layout xml:

```
    <androidx.appcompat.widget.Toolbar
        android:id="@+id/toolbar"
        android:layout_width="match_parent"
        android:layout_height="?android:attr/actionBarSize"
        app:title="Amplify Developer Menu"/>
```